layouts/header.html

```diff
@@ -7,10 +7,10 @@
 
       <nav id="navigation">
         <ul>
```

Comment:
I wasn't sure if you were proposing to remove the nav itself - I've updated the Image href so far.